scripts/apps/master-desk/components/AssignmentsComponent.tsx

```diff
@@ -24,23 +24,27 @@ interface IAssignmentStage {
     name: string;
     color: string;
     code: string;
```

Comment:
is the code still used?

Reply:
Used for unique key for `CardListComponent`